src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,63 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useState } from 'react';
+
+interface TodoItemProps {
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<TodoItemProps> = ({ todo }) => {
+  const { id, title, completed } = todo;
+  const [isPending] = useState(false);
+
+  const checkboxName = `checkbox-${id}`;
+
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames('todo', { completed: completed })}
+    >
+      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
```

Comment:
Don't disable linter

Reply:
i don't know why it's not working properly, I've added htmlFor but still get this error

Reply:
> Don't disable linter

👀